modules/apps_api/lib/apps_api/notification_service.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'active_support/time'
+require 'okta/service'
+
+module AppsApi
+  class NotificationService < Common::Client::Base
+    def initialize
+      @okta_service = Okta::Service.new
+      # should match the timing of the AppsApi::FetchConnections job
+      @time_period = 60.minutes.ago.utc.iso8601
+      @connection_template = Settings.vanotify.template_id.connection_template
+      @disconnection_template = Settings.vanotify.template_id.disconnection_template
+      @notify_client = VaNotify::Service.new
```

Comment:
We have introduced changes to VaNotify service. It requires you to pass api key as argument to the constructor. Please take a look at: https://github.com/department-of-veterans-affairs/vets-api/pull/5962.

Reply:
Ah thanks for posting this! I was just looking at the tests failing and feeling really confused for a second. I've made updates to the code to reference the api_key addition

Reply:
You might want to use lighthouse api key.
Settings.vanotify.services.lighthouse.api_key

It might not be in config/settings.yml - @saman-moshafi-va was supposed to add it today.
However it should be there in devops repo for all real environments.

Reply:
Ah, I see. I've updated it to be correct.